LICENSE.txt

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 1999-2005 The Apache Software Foundation
+   Copyright 1999-2025 The Apache Software Foundation
```

Comment:
```suggestion
   Copyright [yyyy] [name of copyright owner]
```

The `LICENSE.txt` file should be identical to https://www.apache.org/licenses/LICENSE-2.0.txt.

We discussed it in apache/logging-log4net#171